tutorial_sarscov2.md

```diff
@@ -125,106 +149,165 @@ aligner = bwa
 
 Check what will be executed:
 ```bash
-snakemake -s vpipe.snake --dryrun -p --cores 2
+./vpipe --dryrun -p --cores 2
 ```
 
 As it is your first run of V-pipe, this will also generate the sample collection table.
 Check `samples.tsv` in your editor.
 
-Note that the demo files you downloaded have only have reads of length 150.
-V-pipe's defaults parameters are optimized for reads length of 200 ; add the third column in the file:
+Note that the demo files you downloaded only have reads of length 150.
+V-pipe's default parameters are optimized for reads of length 250 ; add the third column in the file:
 
 ```
 SRR10903401	20200102	150
 SRR10903402	20200102	150
 ```
 
 > **Tips:** Always check the content of the `samples.tsv` file.
->  - If you didn't use the [correct structure](#preparing-a-small-dataset) this file might end up empty or some entries might be missing.
->  - You can safely delete it and re-run the `--dryrun` to regenerate it
+>  - If you didn't use the [correct structure](#preparing-a-small-dataset), this file might end up empty or some entries might be missing.
+>  - You can safely delete it and re-run the `--dryrun` to regenerate it.
 
 Run the V-pipe analysis (the necessary dependencies will be downloaded and installed in conda environments managed by snakemake):
 
 ```bash
-snakemake -s vpipe.snake --use-conda -p --cores 2
+./vpipe --use-conda -p --cores 2
 ```
 
+## Output
 
-## Larger dataset
+The Wiki contains an overview of the [output files](https://github.com/cbg-ethz/V-pipe/wiki/output).
```

Comment:
maybe href rather at wiki than at output files?

Reply:
My logic is: wiki manual has multiple page, I am point to the one title "output files", so that is the words in the sentence that I will HREF.
(And search engines used to like this a tiny bit better)